libbeat/cfgfile/list.go

```diff
@@ -137,6 +137,16 @@ func (r *RunnerList) Reload(configs []*reload.ConfigWithMeta) error {
 			} else {
 				r.logger.Errorf("Error creating runner from config: %s", err)
 			}
+
+			// If InputUnitID is not empty, then we're running under Elastic-Agent
+			// and we need to report the errors per unit.
+			if config.InputUnitID != "" {
```

Comment:
Just curious, because I'm not super familiar with this code, why not always report errors per unit, i.e. what is this conditional buying us?

Reply:
Or, put differently, in what scenario / code path do we expect `config.InputUnitID` to be empty?

Reply:
`config.InputUnitID` is empty if it is a standalone Filebeat. Only the Elastic-Agent sets the input unit ID.

My goal here was to keep the behaviour unchanged when running a standalone Filebeat. Maybe I'm being over cautious here, I'll give it some thought.